cmd/serverless-init/cloudservice/appservice.go

```diff
@@ -57,6 +59,5 @@ func (a *AppService) Init() error {
 }
 
 func isAppService() bool {
-	_, exists := os.LookupEnv(RunZip)
-	return exists
+	return config.InAzureAppServices(os.LookupEnv)
```

Comment:
Let's remove this entire function and use `config.InAzureAppServices()` in place of it if possible